### ISSUE TITLE
Packaged 0.1.0 aar contains min sdk 15 in the manifest

### ISSUE THREAD

**Issue by gokhanbarisaker** [square/sqlbrite#26]
Hi,

Although the min sdk specified as 9 in the sqlbrite/src/main/AndroidManifest.xml, the packaged aar contains min sdk 15 in its AndroidManifest.xml. For that reason I am receiving error below

```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : uses-sdk:minSdkVersion 10 cannot be smaller than version 15 declared in library /Users/gokhanbarisaker/Workspace/Android/MarkaVIP/app/build/intermediates/exploded-aar/com.squareup.sqlbrite/sqlbrite/0.1.0/AndroidManifest.xml
  	Suggestion: use tools:overrideLibrary="com.squareup.sqlbrite" to force usage
```

I added the dependency using

```
compile 'com.squareup.sqlbrite:sqlbrite:0.1.0'
```

and I am using jCenter as a maven repository.

I know, I can use *tools:overrideLibrary="com.squareup.sqlbrite"* option as suggested on the error above. Still it just doesn't feel right, also I believe you will feel the same.

Thanks in advance,

**Comment by gabrielittner** [comment]
The min sdk change isn't published yet, the 0.1.1-SNAPSHOT has it already.

**Comment by JakeWharton** [comment]
Yep. We hope to release soon but there's been some things occupying us internally.
